README.md

```diff
@@ -75,14 +100,14 @@ spec:
         app: greeter
     spec:
       containers:
-      - name: greeter
-        image: moleculer/demo:1.4.2
-        livenessProbe:
-          httpGet:
-            path: /live
-            port: 3001
-        readinessProbe:
-          httpGet:
-            path: /ready
-            port: 3001
+        - name: greeter
```

Comment:
Again, stop polluting this pull request which is about a custom liveness checker and not about file formats